extensions/ql-vscode/src/vscode-tests/minimal-workspace/determining-selected-query-test.ts

```diff
@@ -16,20 +16,20 @@ export function run() {
   describe('Determining selected query', async () => {
     it('should allow ql files to be queried', async () => {
       const q = await determineSelectedQuery(Uri.parse('file:///tmp/queryname.ql'), false);
-      expect(q.queryPath).to.equal('/tmp/queryname.ql');
+      expect(q.queryPath).to.equal(path.join('/', 'tmp', 'queryname.ql'));
```

Comment:
Is using `/` really platform agnostic? I guess if tests pass, then it's fine.

Reply:
Yeah, that counts as the root directory even in `path.win32`.